src/styles/components/_home-search-box.scss

```diff
@@ -1,18 +1,65 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  display: flex;
+  width: 80%;
 
-  &__button {
-    @include button-darken-effect($picton-blue);
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px) 0px box-shadow-opacity(0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
 
-    text-decoration: none;
+  &__input {
+    width: 100%;
+    box-sizing: border-box;
     padding: rem(5px 10px);
-    border-radius: rem(3px);
-    background-color: $picton-blue;
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
+
```

Comment:
Do you think we need `text-overflow: ellipsis;` for making `...` in small screen?

**Before**
<img width="259" alt="Screen Shot 2564-03-01 at 15 25 38" src="https://user-images.githubusercontent.com/32285706/109471574-83893f80-7aa3-11eb-9ac2-695687857fd7.png">

**After**
<img width="259" alt="Screen Shot 2564-03-01 at 15 25 29" src="https://user-images.githubusercontent.com/32285706/109471587-871cc680-7aa3-11eb-92c8-ea3d48225b90.png">

Reply:
Very interesting. I'll apply that.